ql/src/test/results/clientnegative/load_wrong_noof_part.q.out

```diff
@@ -6,4 +6,4 @@ POSTHOOK: query: CREATE TABLE loadpart1(a STRING, b STRING) PARTITIONED BY (ds S
 POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@loadpart1
-FAILED: SemanticException [Error 10006]: Line 2:82 Partition not found ''2009-05-05''
+#### A masked pattern was here ####
```

Comment:
How it is related to this change.

Reply:
In HIVE-5601, the file path in the q file was given an incorrect path, although the purpose of the test was to check for number of  partitions. With this PR change that wrong path would throw the semantic exception even before checking the number of partitions.
Instead of aiming for the file path and raising a semantic exception, I have updated the q file with the right file path, fulfilling the task of merely verifying the number of partitions.